src/components/splitButton/splitButton.ts

```diff
@@ -0,0 +1,400 @@
+import { LitElement, html } from 'lit';
+import {
+  customElement,
+  property,
+  state,
+  query,
+  queryAssignedElements,
+} from 'lit/decorators.js';
+import '../icon';
+
+import { classMap } from 'lit/directives/class-map.js';
+import downIcon from '@carbon/icons/es/chevron--down/24';
```

Comment:
The chevron down should be 20px size based on the figma design.

Reply:
yes Robert said that and need to change this.

Reply:
Done